### ISSUE TITLE
in save and demo games health bar are not redrawn on load

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#63]
```
What steps will reproduce the problem?
1. Load a demo or a save
2. the health bar are empty
3. ???
4. profit when one hog is hurt (and bars are redrawn)

What is the expected output? What do you see instead?
the health bars should load with the correct status
```

-----
Original issue reported on code.google.com by `vittorio...@gmail.com` on 25 Sep 2010 at 1:09

**Comment by GoogleCodeExporter** [comment]
```
actually the issue was only for save files
fixed in 4041:fe799b5d601b
```

-----
Original comment by `vittorio...@gmail.com` on 31 Oct 2010 at 5:28
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified (bars there)
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 8:04
* Changed state: **Verified**
